rfq/negotiator.go

```diff
@@ -13,6 +13,12 @@ import (
 	"github.com/lightningnetwork/lnd/routing/route"
 )
 
+// pubKeyFixedBytes is a fixed size byte array that holds a compressed public
+// key.
+//
+// TODO(ffranr): Export this type from the btcec package.
+type pubKeyFixedBytes = [btcec.PubKeyBytesLenCompressed]byte
```

Comment:
Can't we just use `asset.SerializedKey`?

Reply:
That's what I needed, thanks!

That type should probably be moved into `btcec`. Would be easier to find. I didn't think to look in `asset`.

Reply:
Yeah, we have a couple of aliases for the same thing. For example `route.Vertex` in `lnd` and lots of just `[33]byte` types all over. So maybe a type in `btcec` would actually make sense.